src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.Common.targets

```diff
@@ -45,9 +45,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     with the referencing project's target framework.
   ============================================================
    -->
-  <Target Name="GetTargetFrameworkProperties" Returns="TargetFramework=$(NearestTargetFramework);ProjectHasSingleTargetFramework=$(_HasSingleTargetFramework)">
+  <Target Name="GetTargetFrameworkProperties" Returns="TargetFramework=$(NearestTargetFramework);ProjectHasSingleTargetFramework=$(_HasSingleTargetFramework)$(RuntimeIdentifierOverride)">
 
     <PropertyGroup>
+      <!-- If this project is not RID-specific, specify that an empty RuntimeIdentifier should be passed. -->
```

Comment:
Do we actually want to pass an empty RID or is this now just like `ProjectHasSingleTargetFramework` where it should instead be removed?